ui/src/app/views/workspace-edit/component.ts

```diff
@@ -348,10 +359,10 @@ export class WorkspaceEditComponent implements OnInit {
     return Object.keys(input);
   }
 
-  bucketAsTwo(input: Array<string>): Array<Array<string>> {
+  bucketAsThree(input: Array<string>): Array<Array<string>> {
```

Comment:
minor nit: this could be `bucketAsN` and take an int as an argument which would make this more generic and potentially useful as a utility, but it's only used once, so feel free to ignore.

Reply:
I'm going to punt on that, because I think we should do that as soon as we need to have multiple use cases.

Reply:
Yup - I would punt on it too.